ui/apps/platform/src/queries/controls.js

```diff
@@ -218,7 +218,7 @@ export const CONTROL_QUERY = gql`
                     scope
                 }
                 keys {
-                    ... on Node {
+                    ... on ComplianceDomain_Node {
```

Comment:
I guess technically this breaks GraphQL API compatibility, but I guess this is not a concern? Did we never give out scripts with GraphQL API usage to customers?